src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -263,7 +263,7 @@ public void trimToNewStartPosition( final int newStart ) {
             }
 
             // Record our new start boundary
-            cachedInterval = new SimpleInterval(cachedInterval.getContig(), newStart, cachedInterval.getEnd());
+            cachedInterval = new SimpleInterval(cachedInterval);
```

Comment:
Hey there, you've just introduced a bug! Notice that `newStart` is specified for the start position of the new interval.